soperator/test/common/enroot.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+ENROOT_CONFIG_DIR='/root/.config/enroot'
```

Comment:
Should we run test from user root? I implemented running enroot from different users

Reply:
Enroot configuration could be removed at all later, as it's not required to configure it